kernel/src/lib.rs

```diff
@@ -89,6 +89,10 @@
 #![warn(unreachable_pub)]
 #![no_std]
 
+// Define the kernel major and minor versions
+pub const MAJOR: u16 = 2;
+pub const MINOR: u16 = 0;
```

Comment:
We need some way to remember to update this.

Reply:
Could we auto generate/update this from git tags?

Reply:
Generate doesn't work if someone downloads the zip file and there is no git repo. But something that at least checked
somehow would be nice.